.github/workflows/main.yml

```diff
@@ -9,8 +9,7 @@ jobs:
         uses: actions/checkout@master
       - name: Install Package Dependencies
         run: |-
-          Rscript -e "install.packages('remotes', repos = 'https://demo.rstudiopm.com/all/__linux__/bionic/latest')"
```

Comment:
@tevyedavis , please delete these two lines from your .github/workflows/main.yml and it should work again.